lib/vdc/BOVCollection.cpp

```diff
@@ -181,6 +197,20 @@ int BOVCollection::_parseHeader(std::ifstream &header)
 
 int BOVCollection::_validateParsedValues()
 {
+    // Validate that the data file exists.
+    // If not given an absolute path, construct one
+    // from the header file's containing directory.
+    char  actualPath[PATH_MAX + 1];
+    char *success = realpath(_dataFile.c_str(), actualPath);
+    if (success == nullptr) {
+        // At this point we can't find the absolute path, so it might be a relative path.
+        // Try prepending the directory of the .bov file to the data file
+        _dataFile = _currentFilePath + "//" + _dataFile;
+        success = realpath(_dataFile.c_str(), actualPath);
```

Comment:
Did you remember to free the buffer returned by `realpath()`?

Reply:
The return value not a buffer - the buffer (actualPath) is a local array, so it doesn't need to be freed.

https://stackoverflow.com/questions/1563168/example-of-realpath-function-in-c

Reply:
That's right! No need to be freed!